<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMenu</class>
 <widget class="QWidget" name="UserMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_cash">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 255, 255);
color: #616161;
border-radius:20px;
font-size:22px;
border:3px solid #ECEFF1;
}
QPushButton:hover {
background-color: #ECEFF1;
color:#37474F;
}</string>
   </property>
   <property name="text">
    <string>Cash</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_transactions">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 255, 255);
color: #616161;
border-radius:20px;
font-size:22px;
border:3px solid #ECEFF1;
}
QPushButton:hover {
background-color: #ECEFF1;
color:#37474F;
}</string>
   </property>
   <property name="text">
    <string>Transactions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_donation">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 255, 255);
color: #616161;
border-radius:20px;
font-size:22px;
border:3px solid #ECEFF1;
}
QPushButton:hover {
background-color: #ECEFF1;
color:#37474F;
}</string>
   </property>
   <property name="text">
    <string>Donation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_logout">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>470</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #EF5350;
background-image: url(../img/icon_logout.png);
background-repeat: no-repeat;
background-position: right middle;
color: white;
border-radius:20px;
font-size:22px;
}
QPushButton:hover {
background-color: #F44336;
}</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_balance">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 255, 255);
color: #616161;
border-radius:20px;
font-size:22px;
border:3px solid #ECEFF1;
}
QPushButton:hover {
background-color: #ECEFF1;
color:#37474F;
}</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:28px;
color:white;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Teppo Testaaja</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-size:22px;</string>
   </property>
   <property name="text">
    <string>Good afternoon,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_deposit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 255, 255);
color: #616161;
border-radius:20px;
font-size:22px;
border:3px solid #ECEFF1;
}
QPushButton:hover {
background-color: #ECEFF1;
color:#37474F;
}</string>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
